sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_generated_identity/v0_1_preview/aio/_configuration.py

```diff
@@ -0,0 +1,55 @@
+# coding=utf-8
```

Comment:
I'm not sure this client needed to be generated as multiapi, I'm only seeing one version (`v0_1_preview`). There could be reasons I'm missing though, please lmk!

Reply:
Err I just did it this way as most autorest configs seemed to have `multiapi` present, and it seemed like a default option. Presumably we will eventually have other versions of the API too. Is it worth doing the generation without `multiapi` until we have other versions?

Reply:
Yes, we should not do multiapi generation until we really need it. Even when we need multiapi support, the multiapi option of autorest is not necessarly the only option (it's for complex case)

Reply:
Cool, I regenerated after removing `multiapi: true` with some other changes requested on the Swagger side.